assets/css/src/components/entry-content.css

```diff
@@ -67,4 +67,8 @@
 			width: 10rem;
 		}
 	}
+
+	& a:hover, & a:focus {
+		text-decoration: none;
```

Comment:
@emeaguiar I think we need to distinguish between a normal state vs active/hover state ? As we have underline on normal state I added this so we remove the underline on hover/focus state.